app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
-    # write your code here
-    pass
-
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power, average_rating, count_of_ratings):
```

Comment:
Missing type annotation for function argument 'distance_from_city_center', 'clean_power', 'average_rating', 'count_of_ratings'. Missing return type for __init__ function. Line too long (97 > 79 characters), divide by two other's line